.github/scripts/ci_config.py

```
@@ -173,6 +173,13 @@ def run_group_tests(args):
 
     crates = groups[args.group] or []
     failed = []
+    coverage = getattr(args, "coverage", False)
+
+    if coverage:
+        github_output("crate_flags", ",".join(crates))
+        github_group_start("Cleaning coverage state")
+        subprocess.run(["cargo", "llvm-cov", "clean", "--workspace"])
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)